docs/data/hubble/analyst-guide/end-to-end-analysis-example.mdx

```diff
@@ -0,0 +1,201 @@
+---
+title: "End to End Analysis Example"
+sidebar_position: 40
+---
+
+import create_data_source from "/img/hubble/create-data-source.png";
+import bq_connector from "/img/hubble/bq-connector.png";
+import add_contract_data from "/img/hubble/add-contract-data.png";
+import create_report from "/img/hubble/create-report.png";
+import select_data_sources from "/img/hubble/select-data-sources.png";
+import add_pie_chart from "/img/hubble/add-pie-chart.png";
+import pie_contract_data from "/img/hubble/pie-contract-data.png";
+import pie_config from "/img/hubble/pie-config.png";
+import pie_chart_durability from "/img/hubble/pie-chart-durability.png";
+import click_add_data from "/img/hubble/click-add-data.png";
+import add_custom_query from "/img/hubble/add-custom-query.png";
+import add_column_chart from "/img/hubble/add-column-chart.png";
+import column_chart_config from "/img/hubble/column-chart-config.png";
+import resource_column_chart from "/img/hubble/resource-column-chart.png";
+
+This page will guide you through an end-to-end analysis use case with Hubble and [Google Looker Studio](https://cloud.google.com/looker-studio?hl=en)
+
+## Prerequisites
+
+1. Make sure you have connected to Hubble by following the instructions in the [Connecting](./connecting.mdx) page
+2. You have access to [Google Looker Studio](https://cloud.google.com/looker-studio?hl=en)
+3. You have read and understand the general [Best Practices](https://developers.stellar.org/docs/data/hubble/analyst-guide/optimizing-queries#best-practices) for querying BigQuery data
+
+## Create a Dashboard Analyzing Soroban Metrics
+
+### Attach Data Sources to Looker Studio
+
+1. Select `Create --> Data source`
+
+<img src={create_data_source} width="200" />
+
+2. Find and select the `BigQuery Google Connector`
+
+<img src={bq_connector} width="300" />
+
+3. Find the desired tables that you want to connect. For this example you will want to add a data source for:
+   1. `crypto-stellar.crypto_stellar.contract_data`
```

Comment:
nit: I actually think you want to connect to `contract_data_current`. The current analysis counts each individual time a contract entry has changed

Reply:
Will discuss with the new query example brainstorm